lib/alephant/logger_factory.rb

```diff
@@ -0,0 +1,9 @@
+require "alephant/logger/base"
+
+module Alephant
+  module LoggerFactory
+    def self.create(drivers = [])
```

Comment:
We can drop the default parameter value now, as `#create` will always receive at least an empty [] from the splat